### PR TITLE
Update readme to v0.8

### PR DESCRIPTION
**Pull request by eksperimental**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ To use ExDoc in your Mix projects, first add ExDoc as a dependency:
 
 ```elixir
 def deps do
-  [{:earmark, "~> 0.1", only: :dev},
-   {:ex_doc, "~> 0.7", only: :dev}]
+  [{:earmark, "~> 0.1.17 or ~> 0.2", only: :dev},
```
Comment (josevalim) [inline README.md]: It can be "~> 0.2". We should push people to the latest anyway. We just support older versions for convenience.

Comment (eksperimental) [inline README.md]: great! fixed

Comment (milmazz) [inline README.md]: @josevalim @eksperimental So, this means that we can also update the [mix.exs](https://github.com/elixir-lang/ex_doc/blob/master/mix.exs#L14) file.

**Comment by josevalim** [comment]
Let's keep the mix.exs file the same so old users are fine. We shuold push new users to use the latest though.

**Comment by josevalim** [comment]
Sorry, I am drunk. There is no earmark 0.2 yet :(

**Pull request merged**